code-scanning/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Dependency Review",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Code Scanning",
+      "Automation",
```

Comment:
I don't think you should have Automation as categories for this templates. Automation templates are added to the automation folder. Also, move "Dependency Review" to the start of categories list.

Reply:
moved "Dependency Review" to top of `categories` array, and removed "Automation" 👍 thanks!